diff --git a/lib/message.cc b/lib/message.cc
index 9e1005a3..7a635e00 100644
--- a/lib/message.cc
+++ b/lib/message.cc
@@ -73,7 +73,8 @@ static struct maildir_flag_tag flag2tag[] = {
     { 'F', "flagged", false},
     { 'P', "passed",  false},
     { 'R', "replied", false},
-    { 'S', "unread",  true }
+    { 'S', "unread",  true },
+    { 'T', "trashed", false}
 };
 
 /* We end up having to call the destructor explicitly because we had
