From 97e1236fd5f9eca9ce2c0c1820784648ce2c0284 Mon Sep 17 00:00:00 2001
From: Thomas Tuegel <ttuegel@gmail.com>
Date: Tue, 23 Dec 2014 13:02:32 -0600
Subject: [PATCH 2/3] don't build sparse

---
 2.1/src/Makefile | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/2.1/src/Makefile b/2.1/src/Makefile
index 2f49ae3..1ea36e5 100644
--- a/2.1/src/Makefile
+++ b/2.1/src/Makefile
@@ -92,26 +92,24 @@ endif
 
 #$(NAME): $(OBJECTS) 
 
-all:	$(OBJECTS) $(OBJECTS_dense) $(OBJECTS_banded) $(OBJECTS_sparse)  
+all:	$(OBJECTS) $(OBJECTS_dense) $(OBJECTS_banded)
 	ar r $(NAME)  $(OBJECTS)
 	ar r $(NAME_dense) $(OBJECTS_dense)
 	ar r $(NAME_banded) $(OBJECTS_banded)
-	ar r $(NAME_sparse) $(OBJECTS_sparse)
 	@if test -d ../lib/$(ARCH); then \
 	echo "directory $(ARCH) already exists"; \
 	else echo "create directory $(ARCH)" && mkdir ../lib/$(ARCH); fi;
 	mv *.a ../lib/$(ARCH)
-	rm -f $(OBJECTS) $(OBJECTS_dense) $(OBJECTS_banded) $(OBJECTS_sparse) 
+	rm -f $(OBJECTS) $(OBJECTS_dense) $(OBJECTS_banded)
 
 
 clean: 
 	rm ../lib/$(ARCH)/$(NAME)
 	rm ../lib/$(ARCH)/$(NAME_dense)
 	rm ../lib/$(ARCH)/$(NAME_banded)
-	rm ../lib/$(ARCH)/$(NAME_sparse)
 	rm -r ../lib/$(ARCH)
 touch:
-	@touch $(CODE90) $(CODE90_dense) $(CODE90_banded) $(CODE90_sparse)
+	@touch $(CODE90) $(CODE90_dense) $(CODE90_banded)
 
 
 
-- 
2.1.3

