From d9dbeea909777316bbe00b0f9b5a78f642146722 Mon Sep 17 00:00:00 2001
From: Thomas Tuegel <ttuegel@gmail.com>
Date: Fri, 26 Dec 2014 15:20:00 -0600
Subject: [PATCH 3/3] use -fPIC

---
 2.1/src/make.inc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/2.1/src/make.inc b/2.1/src/make.inc
index f8f6c6e..b7531b9 100644
--- a/2.1/src/make.inc
+++ b/2.1/src/make.inc
@@ -15,7 +15,7 @@ OPTION = 1
 ########## Select C compiler and flags (mandatory for option #2)
 ################################################################
 CC     = gcc
-CFLAGS = -O3 -fopenmp
+CFLAGS = -O3 -fopenmp -fPIC
 
 
 ##########################################################
@@ -30,7 +30,7 @@ CFLAGS = -O3 -fopenmp
 
 ##F2- Gnu Fortran Compiler
 F90 = gfortran 
-F90FLAGS = -O3 -m64 -openmp -ffree-line-length-none -ffixed-line-length-none -cpp 
+F90FLAGS = -O3 -m64 -openmp -ffree-line-length-none -ffixed-line-length-none -cpp -fPIC
 
 ##F3-Portland Group Fortran Compiler
 #F90 = pgf90
-- 
2.1.3

